security_service/src/main/java/com/eai/securityservice/controller/OtpEmailController.java

```diff
@@ -0,0 +1,30 @@
+package com.eai.securityservice.controller;
+import com.eai.openfeignservice.user.ClientRequest;
+import com.eai.securityservice.service.CompareOtpEmail;
```

Comment:
You can merge those two services, GenerateOtpEmail and CompareOtpEmail, into a single service called OtpEmailService. Similarly, you can do the same for the phone services.